src/shared/components/palette/complex/fields-filter.pipe.ts

```diff
@@ -82,7 +82,7 @@ export class FieldsFilterPipe implements PipeTransform {
 
     return fields
       .map(f => {
-        let clone = { ...f };
+        let clone = Object.assign({}, f);
```

Comment:
if you look at field.utils.ts class there is a method `cloneObject` that does exactly this. Can you make it `public` and call that here please?